share/completions/attrib.fish

```diff
@@ -0,0 +1,30 @@
+function __attrib_generate_args --description 'Function to generate args'
+  set --local current_token (commandline --current-token --cut-at-cursor)
+
+  switch $current_token
+    case '+*'
+      echo -e 'r\tSet the Read-only file attribute
+a\tSet the Archive file attribute
+s\tSet the System file attribute
+h\tSet the Hidden file attribute
+i\tSet the Not Content Indexed file attribute' | awk -F '\t' "{ printf \"$current_token%s\t%s\n\", \$1, \$2 }"
+    case '-*'
+      echo -e 'r\tClear the Read-only file attribute
+a\tClear the Archive file attribute
+s\tClear the System file attribute
+h\tClear the Hidden file attribute
+i\tClear the Not Content Indexed file attribute' | awk -F '\t' "{ printf \"$current_token%s\t%s\n\", \$1, \$2 }"
+    case '*'
+      if __fish_seen_argument --windows 's'
```

Comment:
If I run this, I get `__fish_seen_argument: --windows: unknown option` - probably this needs to be 
```suggestion
      if __fish_seen_argument -- --windows 's'
```

Reply:
Everything works nice on my machine. :thinking: I've copied completion to my terminal and tested with fake ampty `attrib` executable. 

![image](https://user-images.githubusercontent.com/42812113/150729876-bf485a02-1480-44f5-9af6-93f113901087.png)